apiserver/facades/agent/provisioner/provisioner_test.go

```diff
@@ -1351,11 +1351,12 @@ func (s *withoutControllerSuite) TestContainerManagerConfig(c *gc.C) {
 
 func (s *withoutControllerSuite) TestContainerConfig(c *gc.C) {
 	attrs := map[string]interface{}{
-		"http-proxy":            "http://proxy.example.com:9000",
-		"apt-https-proxy":       "https://proxy.example.com:9000",
-		"allow-lxd-loop-mounts": true,
-		"apt-mirror":            "http://example.mirror.com",
-		"cloudinit-userdata":    validCloudInitUserData,
+		"http-proxy":                   "http://proxy.example.com:9000",
+		"apt-https-proxy":              "https://proxy.example.com:9000",
+		"allow-lxd-loop-mounts":        true,
+		"apt-mirror":                   "http://example.mirror.com",
+		"cloudinit-userdata":           validCloudInitUserData,
+		"container-inherit-properties": "write_files,apt",
```

Comment:
At first read, this doesn't see quite right. If the inherited properties say that the container should use the apt mirror from the host for example, then we should be setting apt-mirror here. Same with apt proxy etc. Or if we are to use the CA certs from the host, then there should be a CA certs entry.

Reply:
updated